### ISSUE TITLE
Stuck on downloading graphics patch?

### ISSUE THREAD

**Issue by ohhaimark345** [07th-mod/higurashi-console-arcs#27]
So I'm using the automatic installer, and I don't know how long it takes for the graphics patch to download, but I think mine might be stuck.  It's been like half an hour and it still hasn't finished, I've tried uninstalling the installer and doing it again, uninstalling the game, and various other things. Any idea what the problem is? Does it just take longer to download (and I'm really impatient?). 
![higurashipatch](https://user-images.githubusercontent.com/40499935/41792562-460a20a8-7627-11e8-85ea-e7325d0dafb8.PNG)

**Comment by ItaloKnox** [comment]
This doesn't look right, the installer should display the download speed and time left. Can you upload the log? It should be inside a folder called ``Logs`` right beside the installer .exe.

**Comment by ohhaimark345** [comment]
Sorry, I don't know where to find the folder you're talking about. All I did was download the 07th Mod Patcher, run it, and start the installation process.  I don't know how to get to the "logs" file.
![hig2](https://user-images.githubusercontent.com/40499935/41794321-01e3ba82-762d-11e8-8d4e-9dfe7f0500f5.PNG)
![hig3](https://user-images.githubusercontent.com/40499935/41794323-01f67abe-762d-11e8-9a79-f4eb8cd363dd.PNG)

**Comment by DoctorDiablo** [comment]
Look in the same folder where you put the 07th-Mod.patcher.  There should be another folder in there called Logs.  It gets created when you run the install.  Can you look in there for the log file and send it to us?

**Comment by ohhaimark345** [comment]
Found it! Sorry about that. Here's a picture. The second one is of what appears when I open the PatcherLog file.
![hig5](https://user-images.githubusercontent.com/40499935/41796491-cbea628e-7634-11e8-9de6-f936fb6f43e4.PNG)

![hig4](https://user-images.githubusercontent.com/40499935/41796446-948bf6f4-7634-11e8-8b20-0c7d9120088d.PNG)

**Comment by ItaloKnox** [comment]
That's interesting, looks like a line ending issue. I'm going to check it right now.

**Comment by ItaloKnox** [comment]
This is a pain in the ass to fix so I passed the message to have it fixed as soon as possible. I'll make sure to update here when it's fixed. For the time being, I recommend just doing the manual installation.

Sorry about that!

**Comment by ItaloKnox** [comment]
A fix for this was pushed just a few moments ago. The installer should work now. Thanks for reporting this!

**Comment by ohhaimark345** [comment]
Thank you! I'll let you know how it works.

**Comment by ohhaimark345** [comment]
So I left my computer alone and let it download for a while and when I got back, Windows was giving me the automated message that it wasn't working and to report the problem. I exited out of it and the installer shut down too, but when I started up the game, everything seems to be fine.  Do you think it's fine to play? I just don't want to be in the middle of the game and have it crash and lose everything lol

**Comment by ItaloKnox** [comment]
That's a little weird, the installer isn't supposed to break like that. If you have the log I will gladly take a look at it and see if there were any issues with the installation. Just drop the .txt file here and it will be uploaded. 👍

**Comment by ohhaimark345** [comment]
Thank you very much!
[PatcherLog.txt](https://github.com/07th-mod/higurashi-console-arcs/files/2130098/PatcherLog.txt)

**Comment by ohhaimark345** [comment]
Here's also a picture of what's inside the log folders itself it that helps you see anything. 
![hig4](https://user-images.githubusercontent.com/40499935/41812170-0ab77936-76ec-11e8-8cb2-31ee88e3978b.PNG)

**Comment by ItaloKnox** [comment]
This log doesn't show anything about the patch even being downloaded. You sure the game was patched? If you were installing the console arcs directly into an already patched version of Himatsubushi, you should delete everything and start from a fresh copy of the game.

We also released a new version of the installer with a debug function and an extra fix for your issue. Can you download it and try again? Only if your game isn't really patched, I mean. If it looks fine, then you should leave it like that. [The installer can be found here.](https://github.com/07th-mod/Higurashi_Installer_WPF/releases/tag/v1.10)

**Comment by ohhaimark345** [comment]
Yeah, I've been playing the game for an hour or so and everything seems to be fine, and I don't wanna mess anything up, so I'll just leave it as it is. Though, I did make sure to delete all my game data and uninstall it from steam before I used the new fixed installer. So it should have been a fresh copy of the game after I redownloaded it from Steam. Once again, thank you for everything.  I've been looking forward to more Higurashi so getting the chance to play these is awesome. I'll be sure to let you know or open up a new issue if anything happens with game or goes wrong.
